packages/action-tracker/src/index.ts

```diff
@@ -15,14 +15,29 @@ import {
 export const buildEmbraceMiddleware = () => {
   return (store: IMiddleware<IDispatch, any>) =>
     (next: IDispatch<IAnyAction>) =>
-    (action: IAnyAction) => {
+    async (action: IAnyAction) => {
       if (!next || typeof next !== "function") {
         console.warn(
           "[Embrace] The state managment middleware was not applied succesfully",
         );
         return 0;
       }
       const startTime = new Date().getTime();
+      const attributes = {
+        name: action.type.toString().toUpperCase(),
+        payload_size: zip(action.payload || 0).length,
```

Comment:
attributes must be strings, I was seeing this show up in the dashboard correctly after casting:

```suggestion
        payload_size: (zip(action.payload || 0).length).toString(),
```